codalab/rest/bundles.py

```diff
@@ -736,7 +746,28 @@ def _update_bundle_contents_blob(uuid):
         if request.query.filename:
             filename = request.query.get('filename', default='contents')
             sources = [(filename, request['wsgi.input'])]
-        if sources:
+
+        if use_azure_blob_beta:
+            local.model.update_bundle(
+                bundle,
+                {
+                    'metadata': {
+                        # TODO(Ashwin): don't hardcode storage account name.
+                        'link_url': f"azfs://storageclwsdev0/bundles/{bundle.uuid}{'/contents.zip' if filename.endswith('.zip') else '/contents'}",
```

Comment:
What's our general solution here?  We should talk about how we want to name these for dev and prod?  I guess we should also document how these are setup in the deployment repo.

Reply:
We need to pass the storage container name in as environment variables (will update to do that).